### PR TITLE
INTL-117: Add a functional test for intl_message_migration

### PR DESCRIPTION
**Pull request by alanknight-wk**
## Motivation
  
I noticed that we never actually removed (or fixed) the sorting by line of output messages, which was a problem for multiline strings. None of our tests actually exercised as far as writing the output file, where this happens.

## Changes
  
Add a functional test, modeled after the mui migration test that actually runs the executable and verifies the output files. Very basic so far.

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Client Platform member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/src/intl_suggestors/utils.dart
Comment (alanknight-wk) [inline lib/src/intl_suggestors/utils.dart:148]: Using tabs makes the matching in the tests painful, and there doesn't seem to be a good reason to prefer it to spaces.

**Comment by alanknight-wk** [comment]
@greglittlefield-wf Maybe you can help with the failure here. I'm trying to add a round-trip test modeled on the mui_migration_test, but I'm getting a Forbidden failure when it tries to do a pub get. So presumably it can't see our internal pub. But I don't know why it would be different between the two tests/executables.

**Comment by greglittlefield-wf** [comment]
Sorry @alanknight-wk for the delay!

Our workaround there was to run tests that depend on `pub get`-ing privately hosted packages to run in Skynet.

This is the commit that set it up: https://github.com/Workiva/over_react_codemod/commit/5bb79f1edc6d1073e737137e12e9ffbcab9d2d6e

tl;dr: if you add `tags: ['wsd']` to your test suite like we do in [mui_migration_test.dart](https://github.com/Workiva/over_react_codemod/blob/master/test/executables/mui_migration_test.dart#L314), it'll automatically get skipped in GH actions and get run in Skynet instead.

Sorry, a better name for that tag probably would have been `uses-workiva-hosted-packages` or something 😓.

**Review by greglittlefield-wf — APPROVED** [review]
Test setup and changes look good to me, but I'm not sure what exactly the sorting behavior is that you guys are going for, so I'll defer to the INTL squad for that part.

**Comment by alanknight-wk** [comment]
QA +1
The only non-test changes are switching from tab indents to spaces, and tweaking the executable help text.

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Review by robbecker-wf — COMMENTED** [review]
I can't apply an approval now that it is merged, but the sorting LGTM.

**Pull request merged**
